src/ontology/mondo-ingest.Makefile

```diff
@@ -11,6 +11,7 @@
 ### Standard constants #############
 ####################################
 MAPPINGSDIR=../mappings
+DOCS_DIR=../../docs/
```

Comment:
## `DOCS_DIR`
Added this. It is in line with a general pattern we use for referencing directories in the `makefile`, and also easier and less error prone than having to remember / type the path.